internal/flowcontrol/flow_control_manager.go

```diff
@@ -61,6 +60,17 @@ func (f *flowControlManager) RemoveStream(streamID protocol.StreamID) {
 	f.mutex.Unlock()
 }
 
+func (f *flowControlManager) UpdateTransportParameters(params *handshake.TransportParameters) {
```

Comment:
I think it would be nicer to remove the handshake pkg dep here and just have the relevant ints as parameters.

Reply:
You're right, but I won't fix this, since the `FlowControlManager` will be gone as soon as I merge #881 anyway.